build.gradle.kts

```diff
@@ -89,7 +90,10 @@ val versionSuffix = if (channel.isBlank()) "" else "-$channel"
 project(":") {
     val clionVersion = prop("clionVersion")
     version = "0.1.0.${prop("buildNumber")}$versionSuffix"
-    intellij { pluginName = "intellij-rust" }
+    intellij {
+        pluginName = "intellij-rust"
+        setPlugins("org.toml.lang:0.0.1.1759")
```

Comment:
I think we should specify a path to Gradle subproject here, like `project(':toml')`.